uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveRequestTest.php

```diff
@@ -561,7 +563,7 @@ public function testCalculateBalanceChangeForALeaveRequestForAContactWithMultipl
     // The work pattern will rotate and use the week 2
 
     // Monday is not a working day on week 2
-    $expectedResultsBreakdown['breakdown'][] = [
+    $expectedResultsBreakdown['breakdown']['2016-08-15'] = [
```

Comment:
Since now it seems that the breakdown is also indexed by date, will this affect the output of the api that also uses this method?

Reply:
I don't think so, since its still basically the same array but will check with @igorpavlov , if it affects anything.